README.md

```diff
@@ -32,6 +32,21 @@ external evaluation engine in a vendor agnostic way.
 
 The OpenFeature project will include client libraries for common technology stacks including, but not limited to:
 
-* Golang
-* Java
-* JavaScript/TypeScript
+- Golang
+- Java
+- JavaScript/TypeScript
```

Comment:
Given we haven't approached the client vs server side yet, do we want to be more constrained here and limit it to NodeJS?

Reply:
That's probably a good idea for now.